### DISCUSSION TITLE
How to use custom PostCSS config?

### DISCUSSION THREAD

**Discussion by nicooprat** [codesandbox/codesandbox-client#3918] [General] [answered]
I'm trying to use Tailwind [as a PostCSS plugin with a custom config](https://tailwindcss.com/docs/installation/). I used `vue-cli-plugin-tailwind` and it runs fine on local. Then I created a repo in Github with this process and imported it in Codesandbox, and in the `<head>` it just outputs something like this:

```html
<style id="/src/assets/tailwind.css:-css" type="text/css">
@tailwind base;
@tailwind components;
@tailwind utilities;
</style>
```

Here's a simple reproduction test: https://codesandbox.io/s/tailwind-codesandbox-issue-tclfy

The only one sandbox that looks like to use it correctly is this one (using Nuxt) but it's marked as an "Experimental Sandbox", whatever it means: https://codesandbox.io/s/o4vn5pvp7q

I also created a discussion in the Tailwind repo since I don't know where the issue comes from: https://github.com/tailwindcss/tailwindcss/discussions/1576

Hope someone knows how to solve this! 🙏

**Comment by garethx** [comment, marked as answer]
We don't support that in client sandboxes at the moment, so you will need to use a container sandbox, of which Nuxt is one, but you can also use a plain Node sandbox too (others include: Angular, Adonis, Gatsby, Marko, Next, Sapper, Apollo, Ember, Nest, Styleguidist, MDX Deck, Gridsome, Quasar, and Vuepress).

Here's more info on the difference between client and container sandboxes on CodeSandbox should it be useful: https://codesandbox.io/docs/environment#what-is-an-environment

**Reply by nicooprat** [reply]
Thanks for the explanation, I will try that. 

Would be great to have this in a FAQ, because there're no errors and I spent a whole day on this one :) Like "how to use PostCSS with Codesandbox?".
